configtx/msp.go

```diff
@@ -139,6 +145,20 @@ func (m *MSP) isCACert(signingCert *x509.Certificate) error {
 	return fmt.Errorf("signing cert is not a root/intermediate cert for this MSP: %s", m.Name)
 }
 
+func (m *MSP) setMSPConfig(configGroup *cb.ConfigGroup) error {
+	mspConfig, err := newMSPConfig(*m)
+	if err != nil {
+		return fmt.Errorf("new msp config: %v", err)
+	}
+
+	err = setValue(configGroup, mspValue(mspConfig), AdminsPolicyKey)
```

Comment:
why won't this work with just `mspValue(m)`?

Reply:
No, `mspValue()` takes a proto.